songs_cho/05_Volsklieder/Guggisberg.txt

```diff
@@ -0,0 +1,59 @@
+{title: Vreneli vo Guggisberg / S isch äben e Mönsch uf Aerde}
+{artist: anonym}
+
+Quelle: https://als.wikipedia.org/wiki/Text:Volkslieder/S_isch_%C3%A4ben_e_M%C3%B6nsch_uf_Aerde
+
+S isch äben e Mönsch uf Aerde, Simelibärg!
+Und ds Vreneli ab em Guggisberg
+Und s Simmes Hans-Joggeli än dem Bärg,
+S´isch äben ä Mönsch uf Aerde, Dass i möcht bi-n-em si
+
+U ma-n-er mir nit wärde, Simelibärg!
+Und s'Vreneli ab em Guggidbärg,
+Und s' Simmes Hans-Joggeli änet dem Bärg,
+U mah-n-er mir nit wärde, Vor Kummer stirbe-n-i.
+
+I mines Buells Garte
+Und s'Vreneli ab em Guggidbärg,
+Und s' Simmes Hans-Joggeli änet dem Bärg,
+I mines Buelis Garte, da stah zweu Bäumeli.
+
+S isch äben e Mönsch uf Aerde, Simelibärg!
+Und ds Vreneli ab em Guggisberg
+Und s Simmes Hans-Joggeli än dem Bärg,
+Das eini treit Muschgate, das andri Nägeli.
+
+Muschgate, die si süessi, Simelibärg!
+Und ds Vreneli ab em Guggisberg
+Und s Simmes Hans-Joggeli än dem Bärg,
+Muschgate, die si süessi, u d'Nägeli si räss.
+
+I gabs mim Lieb z'versueche
+Und ds Vreneli ab em Guggisberg
+Und s Simmes Hans-Joggeli än dem Bärg,
+I gabs mim Lieb z'versueche, daßs miner nit vergäß.
+
+Ha di no nie vergässe
+Und ds Vreneli ab em Guggisberg
+Und s Simmes Hans-Joggeli än dem Bärg,
+Ha di no nie vergässe, ha immer a di dänkt.
+
+Es si numeh zweu Jahre
+Und ds Vreneli ab em Guggisberg
+Und s Simmes Hans-Joggeli än dem Bärg,
+Es si numeh zweu Jahre, daß mi han a di ghänkt.
+
+Dört unden i der Tiefi
+Und ds Vreneli ab em Guggisberg
+Und s Simmes Hans-Joggeli än dem Bärg,
+Dört unden i der Tiefi, da steit es Mühlirad.
+
+Das mahlet nüt als Liebi,
+Und ds Vreneli ab em Guggisberg
+Und s Simmes Hans-Joggeli än dem Bärg,
+Das mahlet nüt als Liebi, die Nacht und auch den Tag.
+
+Das Mühlirad isch broche
+Und ds Vreneli ab em Guggisberg
+Und s Simmes Hans-Joggeli än dem Bärg,
```

Comment:
hier bräuchte es noch ein meta.yml file im folder 05_Volkslieder (siehe https://github.com/ratnanil/Songs#how-do-i-add-a-new-chapter-into-the-book)

Reply:
Hui, doof. Sorry. Genau dafür wäre der Pull-Request: Ein zweiter schaut drauf, bevor man die Änderungen in die "live" Version merged.